crates/tabby/src/routes/chat.rs

```diff
@@ -41,6 +45,9 @@ pub async fn chat_completions(
             };
 
             let content = format!("data: {}\n\n", content);
+            let event = ChatCompletion { completion_id: id, prompt: prompt.clone(), content: content.clone() };
```

Comment:
the logged event should be per request, instead of per chunk.

You might use a string accumulator to collect the output message, e.g
```rust
let output_message = “”

…
output_message += content
…

// use output_message in event
```